index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Baroka'a Portofolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <img class="logo" src="images/image_geometry.png" alt="logo">
+        <img class="toolbar" src="images/ic_menu.png" alt="">
+      </nav>
+      <div class="headline">
+        <h1 class="hero">Irotori Baroka</h1>
+        <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button type="button" class="header-button">Start collaboration</button>
```

Comment:
- [ ] Kindly add the button interactions (enable, hover, etc.) to your button.
![image](https://user-images.githubusercontent.com/62290545/158578218-1c971e73-9c18-45c0-a704-e74d0c71aef8.png)